doc/developer/building.rst

```diff
@@ -9,25 +9,26 @@ Building FRR
 
    static-linking
    building-frr-for-alpine
+   building-frr-for-archlinux
    building-frr-for-centos6
    building-frr-for-centos7
    building-frr-for-centos8
    building-frr-for-debian8
    building-frr-for-debian9
    building-frr-for-fedora
-   building-frr-for-opensuse
    building-frr-for-freebsd9
    building-frr-for-freebsd10
    building-frr-for-freebsd11
    building-frr-for-freebsd13
    building-frr-for-netbsd6
    building-frr-for-netbsd7
    building-frr-for-openbsd6
+   building-frr-for-opensuse
    building-frr-for-openwrt
    building-frr-for-ubuntu1404
    building-frr-for-ubuntu1604
    building-frr-for-ubuntu1804
    building-frr-for-ubuntu2004
-   building-frr-for-archlinux
+   building-frr-for-ubuntu2204
```

Comment:
We should put this here too https://github.com/FRRouting/frr/blob/master/doc/developer/subdir.am.

Reply:
Missed that, thanks, well update shortly